src/pages/steps/Step1.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement } from 'react'
+import { Title, InformationBox } from 'kyb-core'
+import DefaultTemplate from '../templates/DefaultTemplate'
+import DynamicFormattedMessage from '../../components/common/DynamicFormattedMessage'
+
+function Step1(): ReactElement {
+  return (
+    <DefaultTemplate>
+      <div>
+        <Title title="Société">
+          <DynamicFormattedMessage
+            id="step1.tooltip"
+            values={{
+              a: (...chunks: any) => <a href="https://www.google.com">{chunks}</a>
+            }}
+          />
+        </Title>
+        <div>
+          <DynamicFormattedMessage
+            id="step1.sub-title"
+            tag="p"
+            className="mb2"
+          />
+          <DynamicFormattedMessage id="step1.text1" tag="p" className="mb2" />
+          <DynamicFormattedMessage id="step1.text2" tag="p" className="mb2" />
+        </div>
+        <InformationBox variant="blue">
+          <DynamicFormattedMessage
+            id="step1.information-box"
+            tag="p"
+            className="mb2"
+            values={{
+              b: (...chunks: any) => <b>{chunks}</b>,
+              br: <br />,
+            }}
+          />
+        </InformationBox>
+        <div>
+          <DynamicFormattedMessage
```

Comment:
```suggestion
       <p  className="mb2">
          <DynamicFormattedMessage
            id="step1.text3"
            tag="strong"
          />
        </p>
```
This way you can change 
"step1.text3": "<b>Merci de bien vouloir charger les justificatifs suivants</b>" 
to 
"step1.text3": "Merci de bien vouloir charger les justificatifs suivants",

Try not to use the **<b>** tag it's deprecated

Reply:
Done